authors/apps/authentication/urls.py

```diff
@@ -6,6 +6,6 @@
 
 urlpatterns = [
     path('user/', UserRetrieveUpdateAPIView.as_view()),
-    path('users/', RegistrationAPIView.as_view()),
+    path('users/', RegistrationAPIView.as_view(), name="signup"),
```

Comment:
@engjames Nice initiative here but this is not part of your story. We shall surely use this since its more scalable. You should also remove this.